_includes/domain-search.html

```diff
@@ -37,13 +37,28 @@
                 </div>
                 `
                 )
-            } 
-            else {
+            } else if (response.code == "unavailable") {
+                // The unavailable API response contains duplicative language. Omit it and use our hardcoded feedback instead.
```

Comment:
(Q) Is this something that should be fixed in the API, or is it moreso duplicative in the sense that the message is in parity with the prebuilt response?

Reply:
The message is in parity with the pre-built response